azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -163,6 +163,11 @@ private ExecutorService createExecutorInfoRefresherService() {
         new ThreadFactoryBuilder().setNameFormat("azk-refresher-pool-%d").build());
   }
 
+  @Override
+  public Status getStartStatus() {
+    return Status.PREPARING;
```

Comment:
Shall we mark start status as READY for PUSH based method as well?

Reply:
Poll and containerization has steps where ready is changed to preparation. Push doesn't read from the db so there is no point in adding the ready status and then updating to prepare. Hence prepare there should suffice.